src/data_hub/plotting.py

```diff
@@ -204,8 +206,11 @@ def scatter(
     # this updates the value of the slider to the currently
     # hovered point
 
+    if hover_callback_code is None:
+        hover_callback_code = js_code["hover_callback"]
+
     hover_callback = CustomJS(
-        code=js_code["hover_callback"],
+        code=hover_callback_code,
```

Comment:
we can now inject JS code at execution time to figure out why hovering is broken